### PR TITLE
New EF Hub + toc, docfx and breadcrumb toc updates

### PR DESCRIPTION
**Pull request by wadepickett**
[Internal Review](https://review.docs.microsoft.com/en-us/ef/?branch=pr-en-us-2279)

- Add new EF hub page design - index.yml
- Update TOC.yml to point to new index.yml
- Update docfx.json to include index.yml by including all *.yml at root.
- Update breadcrumb/TOC.yml to point to index.yml specifically at root.

### CODE REVIEW

**Comment by ajcvickers** [comment]
@wadepickett Just want to say I'm really looking forward to reviewing this, but I'm a bit behind because I'm playing catch-up after a few days off.

**Comment by smitpatel** [comment]
@wadepickett - Fixed all build errors. Do you want to continue on this PR to get it merged to master?

File: entity-framework/index.yml [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    # Card
+    - title: "Supported Databases"
+      itemType: overview
+      url: core/providers/index.md
+    # Card
+    - title: "Choose a version of EF that is right for you"
+      itemType: get-started
+      url: efcore-and-ef6/index.md
+    # Card
+    - title: "ASP Reference browser"
```
Comment (smitpatel) [inline entity-framework/index.yml:46]: This title should be updated

**Comment by wadepickett** [comment]
@ smitpatel, many thanks!!!  Please hold off on merging to master, I first I also need sign-off from Robert Lyon who is watching over all Hub designs to make sure they all work together well across products.  I sent him a review request.   I did make the fixes you and Arthur requested.

**Comment by wadepickett** [comment]
Robert Lyon also signed off now so I will go ahead and merge.  Thanks everyone for the fantastic help.

**Pull request merged**